src/forms/registration-popup/index.jsx

```diff
@@ -99,8 +103,8 @@ const RegistrationForm = () => {
     if (!userPipelineDataLoaded && thirdPartyAuthApiStatus === COMPLETE_STATE) {
       if (thirdPartyAuthErrorMessage) {
         setErrorCode(prevState => ({ type: TPA_AUTHENTICATION_FAILURE, count: prevState.count + 1 }));
-        // clear marketingEmailOptIn from local storage if pipeline fails
         localStorage.removeItem('marketingEmailOptIn');
+        localStorage.removeItem('ssoPipelineRedirectionDone');
```

Comment:
Why are we removing this for TPA error message?

Reply:
this is an extra protection. in case of of SSO authentication failed, we are clearing it because user will now perform now SSO authentication.